controllers/storagecluster/placement.go

```diff
@@ -18,7 +18,13 @@ func getPlacement(sc *ocsv1.StorageCluster, component string) rookCephv1.Placeme
 	if ok {
 		(&in).DeepCopyInto(&placement)
 	} else {
-		in := defaults.DaemonPlacements[component]
+		// fallback to 'noobaa-core' if 'noobaa-standalone' placement not found, or use default placement.
+		if component == "noobaa-standalone" {
```

Comment:
I don't think we should add this logic here, This is a generic func, We should check in the calling site and call accordingly.

Reply:
Thanks, sure will update accordingly.